src/main/java/io/candydoc/ddd/extract_ddd_concepts/DDDConceptsExtractionService.java

```diff
@@ -111,24 +125,28 @@ public void apply(CoreConceptFound event) {
     this.handle(CheckConceptInteractions.builder().conceptName(event.getCanonicalName()).build());
   }
 
-  public void apply(InteractionBetweenConceptFound event) {}
-
-  public void apply(ValueObjectFound event) {
+  public void apply(DomainCommandFound event) {
     this.handle(CheckConceptInteractions.builder().conceptName(event.getCanonicalName()).build());
   }
 
   public void apply(DomainEventFound event) {
     this.handle(CheckConceptInteractions.builder().conceptName(event.getCanonicalName()).build());
   }
 
-  public void apply(DomainCommandFound event) {
-    this.handle(CheckConceptInteractions.builder().conceptName(event.getCanonicalName()).build());
+  public void apply(SharedKernelFound event) {
+    this.handle(ExtractCoreConcepts.builder().packageToScan(event.getPackageName()).build());
+    this.handle(ExtractValueObjects.builder().packageToScan(event.getPackageName()).build());
+    this.handle(ExtractDomainEvents.builder().packageToScan(event.getPackageName()).build());
+    this.handle(ExtractDomainCommands.builder().packageToScan(event.getPackageName()).build());
+    this.handle(ExtractAggregates.builder().packageToScan(event.getPackageName()).build());
```

Comment:
Absolutely not.

A Shared Kernel can only contains ValueObjects (and CoreConcepts).

Command and events comes from an aggregate and an aggregate is always in a domain bounded context.